src/components/editCard/EditPreferenceSec.tsx

```diff
@@ -0,0 +1,83 @@
+import { Col, Icon, Row } from "antd";
+import * as React from "react";
+import {ApartmentType, TenantSpecs} from "../../state/profile";
+
+interface PreferenceSecProps {
+  specs: TenantSpecs;
+}
+
+export function EditPreferenceComponent(props: PreferenceSecProps) : JSX.Element {
+  const checkIconStyle = { fontSize: 20, color: "#339a5c" };
+  const crossIconStyle = { fontSize: 20, color: "#b73939" };
+  return (
+    <Row>
```

Comment:
Please break this JSX element down, so that it becomes easier to read. No need to create more files. Just consts that describe what each segment is. It will help readability a lot. Thanks